package.json

```diff
@@ -21,5 +22,13 @@
   "bugs": {
     "url": "https://github.com/oxenprogrammer/thenextweb-clone/issues"
   },
-  "homepage": "https://github.com/oxenprogrammer/thenextweb-clone#readme"
+  "homepage": "https://github.com/oxenprogrammer/thenextweb-clone#readme",
+  "dependencies": {
```

Comment:
These should be `dev dependencies`. Read more here about the difference https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee

Reply:
Thank you for the review, now the dev dependencies have been updated.